pkg/operator/sync.go

```diff
@@ -39,24 +39,35 @@ type syncFunc struct {
 	fn   func(config *renderConfig) error
 }
 
+type syncErr struct {
+	task string
+	error
+}
+
 func (optr *Operator) syncAll(syncFuncs []syncFunc) error {
 	if err := optr.syncProgressingStatus(); err != nil {
-		return fmt.Errorf("error syncing progressing status: %v", err)
+		return syncErr{
+			task:  "ProgressingStatus",
+			error: fmt.Errorf("error syncing progressing status: %v", err),
+		}
```

Comment:
is this bubbled up in the degraded? I think this just just return the error it was returning before

Reply:
oh right. forgot to rm this one again